x/evm/module.go

```diff
@@ -144,6 +144,10 @@ func (am AppModule) RegisterServices(cfg module.Configurator) {
 	if err := cfg.RegisterMigration(types.ModuleName, 5, m.Migrate5to6); err != nil {
 		panic(err)
 	}
+
+	if err := cfg.RegisterMigration(types.ModuleName, 16, m.Migrate16to17); err != nil {
+		panic(err)
```

Comment:
nit: This should be the versioning of the migrations instead of the overall Evmos version.